src/style.css

```
@@ -1 +1,64 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+}
+
+.header {
+  margin: auto;
+}
+
+.header-container {
+  max-width: 1200px;
```

Is it should be here?
```suggestion
  /* max-width: 1200px; */
```